function_calling/notebooks/function_calling_guide.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
I sometimes get this deprecation warning:

```
LangChainDeprecationWarning: The method `BaseTool.__call__` was deprecated in langchain-core 0.1.47 and will be removed in 0.3.0. Use invoke instead.
  warn_deprecated(
```

Could we align with the new `langchain`?

Reply:
updated

Reply:
added invoke in execution method